edxval/tests/test_api.py

```diff
@@ -951,76 +966,108 @@ def get_child_tags(elem):
+
     @data(
         {'course_id': None, 'image': ''},
         {'course_id': 'test-course', 'image': 'image.jpg'},
     )
     @unpack
     def test_basic(self, course_id, image):
+        """
+        Test that video export works as expected.
+        """
         expected = self.parse_xml("""
             <video_asset client_video_id="Shallow Swordfish" duration="122.0" image="{image}">
                 <encoded_video url="http://www.meowmix.com" file_size="11" bitrate="22" profile="mobile"/>
                 <encoded_video url="http://www.meowmagic.com" file_size="33" bitrate="44" profile="desktop"/>
                 <encoded_video url="https://www.tmnt.com/tmnt101.m3u8" file_size="100" bitrate="0" profile="hls"/>
                 <transcripts>
-                    <transcript file_format="sjson" file_name="edxval/tests/data/wow.sjson" language_code="de" provider="3PlayMedia" video_id="{video_id}"/>
-                    <transcript file_format="srt" file_name="wow.srt" language_code="en" provider="Cielo24" video_id="{video_id}" />
+                    <transcript file_format="sjson" file_name="edxval/tests/data/wow.sjson" language_code="de" provider="3PlayMedia" />
+                    <transcript file_format="srt" file_name="edxval/tests/data/The_Flash.srt" language_code="en" provider="Cielo24" />
                 </transcripts>
             </video_asset>
-        """.format(image=image, video_id=constants.VIDEO_DICT_FISH['edx_video_id']))
+        """.format(image=image))
 
         self.assert_xml_equal(
-            api.export_to_xml([constants.VIDEO_DICT_FISH['edx_video_id']], course_id),
+            api.export_to_xml(constants.VIDEO_DICT_FISH['edx_video_id'], self.file_system, STATIC_DIR, course_id),
             expected
         )
 
-    def test_unknown_video(self):
-        with self.assertRaises(ValVideoNotFoundError):
-            api.export_to_xml(["unknown_video"])
-
-    def test_with_multiple_video_ids(self):
+    def test_transcript_export(self):
         """
-        Verify that transcript export with multiple video ids is working as expected.
+        Test that transcript are exported correctly.
         """
-        video_ids = ['super-soaker', 'external_video_id']
-        expected = self.parse_xml("""
-            <video_asset client_video_id="Shallow Swordfish" duration="122.0" image="">
-                <encoded_video bitrate="22" file_size="11" profile="mobile" url="http://www.meowmix.com" />
-                <encoded_video bitrate="44" file_size="33" profile="desktop" url="http://www.meowmagic.com" />
-                <encoded_video bitrate="0" file_size="100" profile="hls" url="https://www.tmnt.com/tmnt101.m3u8" />
+        language_code = 'en'
+        video_id = constants.VIDEO_DICT_FISH['edx_video_id']
+        transcript_files = {'de': u'super-soaker-de.sjson', 'en': u'super-soaker-en.srt'}
+        expected_transcript_path = self.temp_dir + STATIC_PATH + '/{transcript_file_name}'
```

Comment:
Use either str.format OR `os.path.join`

Reply:
`self.temp_dir ` is an absolute path, so `os.path.join(self.temp_dir, STATIC_PATH )` would only yield `/static'. 

Using the `fs.path.combine` which is similar method to `os.path.join` for fs.